### PR TITLE
Update administering-a-multi-geo-environment.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Yammer to the list

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/926

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: Enterprise/administering-a-multi-geo-environment.md [hunk 1]
```diff
@@ -99,3 +99,7 @@ However, Stream is not yet available in the following regions that are currently
 - Canada
 - India
 - United Kingdom
+
+## Yammer
+
+Yammer created for the satellite location will use the end point located in the central location for the tenant. Yammer is not a Multi-Geo service.
```
Comment (JohanFreelancer9) [inline Enterprise/administering-a-multi-geo-environment.md:105]:
```suggestion
Yammer created for the satellite location will use the endpoint located in the central location for the tenant. Yammer is not a Multi-Geo service.
```

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Comment by MikePlumleyMSFT** [comment]
@AdwoodMicrosoft can you [review](https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/pull/947/files)?

**Pull request merged**